pnpmfile.js

```diff
@@ -0,0 +1,16 @@
+//This is needed so pnpm links the needed webpack loaders in node_modules
```

Comment:
What webpack loaders? Do we use webpack?

Reply:
Copy & paste error